docs/observatorium/loki/README.md

```diff
@@ -0,0 +1,52 @@
+## Loki Access
+
+Loki access is open to public and is divided into 3 different components:
+1. Loki Distributor
+2. Loki Frontend
+3. Grafana
+
+While pushing logs into Loki or while querying Loki,
+you will need to have a header `'X-Scope-OrgID'` set
+in your requests. <br>
+The value for this header can be anything.
```

Comment:
Are there any implications of settings this header, just wondering in case that may help inform teams what/how they should pick their orgid.

Reply:
This header is just to identify tenants in a multi tenancy setup ([More info](https://grafana.com/docs/loki/latest/operations/multi-tenancy/)). I am hopig to add a proxy or some other kind of authentication for this, but right now at @durandom's suggestion I am keeping it open so anyone can push logs in.

Reply:
Okay great, it seems like it also plays a role in adding the data source later in the docs. What about adding something like: 

"This OrgID is used to identify tenants in a multi tenancy setup ([More info](https://grafana.com/docs/loki/latest/operations/multi-tenancy/)), it will also play a role when adding a Loki Grafana datasource, more on that later".

Reply:
It seems like you delve more into the OrgID in the `add_loki_grafana_datasource.md`, I think it's sufficient to just forward them here for more info. Which then elaborates further.

Reply:
I added one line that you said to explain what the header does